Source/Core/DolphinQt2/Config/Mapping/Hotkey3D.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
All of these hotkey panes are a huge amount of duplicated code. Can you de-duplicate it somehow?